INSTALL-ADVANCED.md

```diff
@@ -148,3 +148,15 @@ To use a remote Docker Engine (daemon) or balenaEngine, specify the remote machi
 port number with the `--dockerHost` and `--dockerPort` command-line options. For more details,
 check `balena help build` or the [online
 reference](https://www.balena.io/docs/reference/cli/#cli-command-reference).
+
+## Docker Installation
+
+If you're working with Linux and the NPM Installation method doesn't suit your needs
+you may want to try deploying the CLI with Docker.
+
+Official images are not available yet but check out [this guide](docker/DOCKER.md) on how
```

Comment:
So just to confirm, the first step for the user would be to `git clone` the CLI repo, right?

Reply:
I suppose so, or downloading the source tarball or just copying the Dockerfiles? What do we normally suggest in these DIY cases?